include/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Model/Kepler/COE.hpp

```diff
@@ -252,6 +252,30 @@ class COE
     /// @return COE
     static COE FromSIVector(const Vector6d& aCOEVector, const AnomalyType& anAnomalyType);
 
+    /// @brief Construct a frozen orbit from an incomplete set of COEs
```

Comment:
Worth updating the docstrings here to specify what the various combinations of legal inputs are